doc/site/docs/01-getting-started/02-hello-world-example.md

```diff
@@ -5,43 +5,69 @@ sidebar_label: 🌟 Hello, World! Example
 
 # Hello, World! Example
 
-Create `bin/hello_world.dart` and start a basic Relic server:
+Let's get your first Relic server up and running! This guide walks you through creating a minimal "Hello, World!" API to demonstrate just how easy it is to get started with Relic.
 
-```dart
+This example will show you how to:
+
+- Initialize a simple Relic server
+- Define your first route
+- Start the server and handle web requests
+
+### Create the App File
+
+Create a file at `bin/hello_world.dart` with the following content:
+
+```dart file="hello_world.dart"
+import 'dart:developer';
 import 'dart:io';
 import 'package:relic/io_adapter.dart';
 import 'package:relic/relic.dart';
 
 Future<void> main() async {
   // Create a simple handler that responds to every request
-  final handler = respondWith(
-    (final request) => Response.ok(
-      body: Body.fromString('Hello, Relic!'),
-    ),
-  );
+  final router = RelicRouter()
+    ..get('/', (final ctx) => ctx.respond(
```

Comment:
If you want it to respond to every request, regardless of path, then you should use a dynamic tail-match, ie. `/**`, and `.any` instead of `.get`. This will only reply to GET requests for /